### PR TITLE
Use "scheduler" terminology instead of "data set" or "working set"

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

File: tools/crm_simulate.c [hunk 1]
```diff
@@ -517,7 +517,7 @@ main(int argc, char **argv)
     scheduler = pe_new_working_set();
     if (scheduler == NULL) {
         rc = ENOMEM;
-        g_set_error(&error, PCMK__RC_ERROR, rc, "Could not allocate working set");
+        g_set_error(&error, PCMK__RC_ERROR, rc, "Could not allocate scheduler data");
```
Comment (nrwahl2) [inline tools/crm_simulate.c:520]: over 80 chars (already was)

File: include/crm/pengine/internal.h [hunk 2]
```diff
@@ -49,16 +49,16 @@ pcmk_resource_t *pe__last_group_member(const pcmk_resource_t *group);
 #  define pe_proc_err(fmt...) { was_processing_error = TRUE; crm_err(fmt); }
 #  define pe_proc_warn(fmt...) { was_processing_warning = TRUE; crm_warn(fmt); }
 
-#define pe__set_working_set_flags(working_set, flags_to_set) do {           \
-        (working_set)->flags = pcmk__set_flags_as(__func__, __LINE__,       \
+#define pe__set_working_set_flags(scheduler, flags_to_set) do {             \
```
Comment (nrwahl2) [inline include/crm/pengine/internal.h:52]: We could rename these macros themselves since they're internal

**Review by nrwahl2 — COMMENTED** [review]
Looks good :)

File: include/crm/pengine/internal.h (hunk 2 above)
Comment (kgaillot) [reply include/crm/pengine/internal.h:52]: Yep, later PR

File: tools/crm_simulate.c (hunk 1 above)
Comment (kgaillot) [reply tools/crm_simulate.c:520]: Thanks

**Review by nrwahl2 — APPROVED** [review]

**Pull request merged**